frontend/tsconfig.json

```diff
@@ -0,0 +1,38 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "outDir": "build",
+    "module": "esnext",
+    "target": "ES2017",
+    "lib": [
+      "es6",
+      "dom"
+    ],
+    "sourceMap": true,
+    "allowJs": true,
+    "jsx": "react",
+    "rootDir": "src",
+    "moduleResolution": "node",
+    "allowSyntheticDefaultImports": true,
+    "noEmitOnError": false,
+    "pretty": true,
+    "forceConsistentCasingInFileNames": false,
+    "noErrorTruncation": true,
+    "noImplicitReturns": true,
+    "noImplicitAny": false,
+    "noImplicitThis": true,
+    "strictNullChecks": false,
+    "suppressImplicitAnyIndexErrors": true,
+    "resolveJsonModule": true,
+    "noUnusedParameters": false
```

Comment:
From what I see in the code, the frontend was coded the Rust way so unused are marked with `_`.  So instead of allowing unused, we should only allow unused that start with a `_`. I think the old code should pass this way and we prevent regressions in new code.

Reply:
the moment i opened the project i saw a strongly typed background of a person who did this. the flag was deprecated.

Reply:
Setting `"noUnusedParameters": true` requires the following changes:
- AfgHandling.ts:L116
- AfgHandling.ts:L139
- AfgHandling.ts:L195
- App.tsx:L194
- Consensus.tsx:360
- Sparkline.tsx:82